public/video-ui/src/services/VideosApi.js

```diff
@@ -121,12 +121,15 @@ export default {
 
     function updateArticleField(stage, data, composerUrl, pageId) {
       if (data.value || data.belongsTo === 'settings') {
+        const value = data.isFreeText
+          ? data.value.split('"').join('\\"')
```

Comment:
`.replace(/"/g, '\\"')` should work, but its a minor point, so feel free to ignore 😄